### PR TITLE
Add ability to send aspect ratio with send_image and send_images

### LINKED ISSUE #475: send_image and send_images should support aspect ratio parameter
It looks like there was a break in the UI causing images sent via api (not the web client) to use a fallback aspect ratio and object-cover causing different size aspect ratios to render incorrectly: 

image

When you post through the web client, an aspectRatio is set:

image

Should the send_image function include the ability to set aspect ratio?

image

### PR DESCRIPTION
**Pull request by palmer-cl**
- Adds ability to send aspect_ratio with send_images and send_image
- Updated docs to add example using the models

closes #475

### CODE REVIEW

File: examples/send_image.py [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-    client.login('my-handle', 'my-password')
+    client.login("my-handle", "my-password")
 
     # replace the path to your image file
-    with open('cat.jpg', 'rb') as f:
+    with open("cat.jpg", "rb") as f:
         img_data = f.read()
 
-    client.send_image(text='Post with image from Python', image=img_data, image_alt='Text version of the image (ALT)')
+    aspect_ratio = models.AppBskyEmbedDefs.AspectRatio(height=100, width=100)
```
Comment (MarshalX) [inline examples/send_image.py:12]: let's cover this line with some comments

File: examples/send_image.py [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
 
 
 def main() -> None:
     client = Client()
-    client.login('my-handle', 'my-password')
+    client.login("my-handle", "my-password")
 
     # replace the path to your image file
-    with open('cat.jpg', 'rb') as f:
+    with open("cat.jpg", "rb") as f:
```
Comment (MarshalX) [inline examples/send_image.py:9]: pls back single ones

**Review by MarshalX — COMMENTED** [review]
thank you! let's do not touch README.md at all. example in `examples/send_image.py` is enough

File: packages/atproto_client/client/client.py [hunk 3]
```diff
@@ -179,6 +200,7 @@ def send_images(
         text: t.Union[str, TextBuilder],
         images: t.List[bytes],
         image_alts: t.Optional[t.List[str]] = None,
+        image_aspect_ratios: t.Optional[t.List['models.AppBskyEmbedDefs.AspectRatio']] = None,
```
[the remark this answers is not in the document]
Comment (palmer-cl) [reply packages/atproto_client/client/client.py]: Yep. Good call. Updated.

File: examples/send_image.py (hunk 1 above)
Comment (palmer-cl) [reply examples/send_image.py:14]: Added comments.

File: examples/send_image.py (hunk 2 above)
Comment (palmer-cl) [reply examples/send_image.py]: done. forgot to run the formatter.

**Comment by palmer-cl** [comment]
> @palmer-cl pls use ruff as formatter. there is bunch of unexpected formatting changes

@MarshalX see output after running ruff format

image

**Comment by MarshalX** [comment]
`poetry run ruff check .`?

**Comment by MarshalX** [comment]
ah, sorry. not the linter staff. diference in sync and async client

**Comment by palmer-cl** [comment]
Ok I saw the missing comment. Added that in. Looks like that diff was reporting something towards the top of the file but it looks like a formatting diff? Ln 20-23 in your screenshot above?

**Review by MarshalX — APPROVED** [review]

**Comment by palmer-cl** [comment]
@MarshalX are we able to merge this? I know a few people waiting on this functionality. Thanks!

**Comment by MarshalX** [comment]
sure. gonna release soon

**Pull request merged**